WooCommerce/src/main/kotlin/com/woocommerce/android/ui/products/ProductDetailViewModel.kt

```diff
@@ -1038,6 +1063,41 @@ class ProductDetailViewModel @AssistedInject constructor(
         checkImageUploads(getRemoteProductId())
     }
 
+    /**
+     * Finished selecting images for add product flow
+     */
+    @Suppress("unused")
+    @Subscribe(threadMode = ThreadMode.MAIN)
+    fun onEvenMainThread(event: OnAddProductImagesSelectedCompletedEvent) {
+        val originalList = productImagesViewState.localSelectedUriImages?.toMutableList() ?: mutableListOf()
+        originalList.addAll(event.images)
+        viewState = viewState.copy(addProductLocalUris = originalList)
+        productImagesViewState = productImagesViewState.copy(
+            localSelectedUriImages = originalList
+        )
+    }
+
+    fun transformToProductImages(uploadingImageUris: List<Uri>?): List<Product.Image> {
+        productAddImages = uploadingImageUris?.let { list ->
+            list.map { uri ->
+                val tempId =
+                    Random.nextLong(
```

Comment:
Just curious if this is necessary? Would setting it to 0 not work?

Reply:
Ah yes, I think it will. I was trying to be extra safe and make explicit, but if we're happy for it to be `0L` here, I can do that 👌